manifests/init.pp

```diff
@@ -81,6 +85,8 @@
   $profile          = $jboss::params::profile,
   $prerequisites    = Class['jboss::internal::prerequisites'],
   $fetch_tool       = $jboss::params::fetch_tool,
+  $server_opts      = $jboss::params::server_opts,
```

Comment:
I really don't like idea that you add additional parameters to main `jboss` class. They are used only in standalone mode, but they are added to most common class. Maybe you can move them to some other class ex.: `jboss::standalone`?